packages/shared/lib/services/sync/manager.service.ts

```diff
@@ -289,7 +289,7 @@ export class SyncManagerService {
             for (const { syncName, syncVariant } of syncs) {
                 const sync = await getSync({ connectionId: connection.id, name: syncName, variant: syncVariant });
                 if (!sync) {
-                    throw new Error(`Sync "${syncName}" doesn't exists.`);
+                    throw new Error(`Sync "${syncName}" doesn't exists.`); // Todo: return this error instead of throwing
```

Comment:
Why is this a todo instead of just making the change?

Reply:
because it has more side effects or at least it's touching more surface than this PR